src/sphinx_book_theme/nodes.py

```diff
@@ -0,0 +1,48 @@
+from docutils import nodes
+from sphinx.application import Sphinx
+from typing import Any, cast
+from sphinx.writers.latex import LaTeXTranslator
+
+
+class SideNoteNode(nodes.Element):
+    """A node that will not be rendered."""
+
+    def __init__(self, rawsource="", *children, **attributes):
+        super().__init__("", **attributes)
+
+    @classmethod
+    def add_node(cls, app: Sphinx) -> None:
+        add_node = cast(Any, app.add_node)  # has the wrong typing for sphinx<4
+        add_node(
+            cls,
+            override=True,
+            html=(visit_SideNoteNode, depart_SideNoteNode),
+            latex=(visit_SideNoteNode, depart_SideNoteNode),
+        )
+
+
+def visit_SideNoteNode(self, node):
+    if isinstance(self, LaTeXTranslator):
```

Comment:
Maybe this should be "if `not` <html translator of any kind>`"? I assume we'd want other non-HTML builds to not include these either right?

Reply:
At present, I am calling this only for HTML, latex builds https://github.com/executablebooks/sphinx-book-theme/pull/523/files#diff-c428944db19a77ff68c5777783e4a45aa9753788f3620049d9128cab09232611R19, so other builds won't be accessing this function I reckon.